modules/transactionpool/consts.go

```diff
@@ -68,7 +68,7 @@ var (
 var (
 	// minEstimation defines a sane minimum fee per byte for transactions.  This
 	// will typically be only suggested as a fee in the absence of congestion.
-	minEstimation = types.SiacoinPrecision.Div64(100).Div64(1e3)
+	minEstimation = types.SiacoinPrecision.Div64(2e6)
```

Comment:
At this minimum, it costs 1 siacoin to fill a block, or about 3 cents. Given that we've already had people propose adding things like messaging protocols on top of the Sia blockchain using the arbitrary data, I don't think we want to drop the default fee so low. The previous minimum, 60 cents to fill a block, is already really low.

That means a full set of 50 contracts would cost about 20 cents. That seems fine. Even if the price goes up 10x ($10 billion market cap), we're still only at $2 for a full set of contracts.